### ISSUE TITLE
MZIP Exports need to include resources

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/meme#88]
Currently the `*.MEME.ZIP` files exported from the Electron app only contain the runtime loki files.  Resources need to be included as well.

**Comment by benloh** [comment]
@scott.campion I verified that the MZIP export from Electron does not include resource files.

**Comment by benloh** [comment]
NOTE that MZIP is really only used for temporary project loads to review student work and not intended to be a permanent full import.

**Comment by jdanish** [comment]
Agreed and in truth we haven't really been using it - we just save backups of the app and actual data, so maybe if we are moving toward a more robust / dynamic file based approach we should deprecate this?
